tests/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "centos65"
+  config.vm.box_url = "http://downloads.juwai.io/devops/boxes/centos65-x86-64.box"
```

Comment:
I don't know why we have our own box in the first place. Do we want people to use it?